README.rst

```diff
@@ -26,6 +26,11 @@ Installing
     pip install boxsdk
 
 
+
+.. contents:: :depth: 1
```

Comment:
Since we want the SDK to be super-easy to get started with, and since the installation instructions are short, I thought, from a usability perspective, it'd be nice to have the installation instructions to appear prominently at the top, before the lengthy TOC.

Let me know if you'd like me to move it up.